libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3310,23 +3350,35 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 					toCategory == Type::Category::Integer ?
 					dynamic_cast<IntegerType const&>(_to) :
 					addressType;
-
```

Comment:
I think it'd be nice to refactoring this else branch. It's not obvious what `_from` and `_to` are anymore.